### PR TITLE
Add more sections to homepage

### PR DESCRIPTION
**Pull request by basiima**
Add following sections:
- speakers
- schedule
- venue
- sponsors
- newsletter
- buy tickets
- contact us

### CODE REVIEW

File: components/ContactUsSection.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                                <i className="bi bi-envelope"></i>
+                                <h3 className="text-lg uppercase font-bold">Email</h3>
+                                <p><a href="mailto:info@example.com">pyconug@pycon.org</a></p>
+                            </div>
+                        </div>
+
+                    </div>
+                    <form method="POST" action="#" className="block justify-center space-y-8 items-center mb-10">
+                        <div>
+                            <label for="email" className="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300">Your Email</label>
```
Comment (HassanBahati) [inline components/ContactUsSection.js:39]:
```suggestion
                            <label htmlFor="email" className="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300">Your Email</label>
```

File: components/ContactUsSection.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                            <input type="email" className="shadow-sm bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-primary-500 block" placeholder="name@email.com" required />
+                        </div>
+                        <div>
+                            <label for="subject" className="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300">Your Email</label>
```
Comment (HassanBahati) [inline components/ContactUsSection.js:43]:
```suggestion
                            <label htmlFor="subject" className="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300">Your Email</label>
```

File: components/ContactUsSection.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                            <input type="text" className="shadow-sm bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-primary-500 block" placeholder="Let us know how we can help" required />
+                        </div>
+                        <div className="sm:col-span-2">
+                            <label for="message" className="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300">Your Email</label>
```
Comment (HassanBahati) [inline components/ContactUsSection.js:47]:
```suggestion
                            <label htmlFor="message" className="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300">Your Email</label>
```

File: components/SpeakerSection.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+                <div className="mx-auto max-w-2xl text-center">
+                    <h2 className="text-4xl text-[#0e1b4d] font-bold uppercase mb-10">Event Speakers</h2>
+                    <p className="text-base p-0 font-medium">We shall start with guidelines for the Call For Proposals(CFP). Peruse through so that you can prepare before we officially open submissions!<br />This will be a formal invitation for individuals or organizations to submit proposals to speak at the Pycon Uganda conference. It allows conference organizers to identify a diverse range of potential speakers with unique perspectives and experiences. This will help to ensure that the conference features a well-rounded program that is engaging and relevant to attendees. This will also be crucial in promoting diversity and inclusion in the conference program and encourage the sharing of new ideas and approaches.</p>
+                <h6 className="text-center text-[#0e1b4d]">You can view our <a class="buy-tickets scrollto" href="/assets/docs/Call_For_Proposals.pdf" target="_blank" className="text-[#0e1b4d] ml-0">Call For Proposals</a> guidelines.</h6>
```
Comment (HassanBahati) [inline components/SpeakerSection.js:8]:
```suggestion
                <h6 className="text-center text-[#0e1b4d]">You can view our <a className="buy-tickets scrollto" href="/assets/docs/Call_For_Proposals.pdf" target="_blank" className="text-[#0e1b4d] ml-0">Call For Proposals</a> guidelines.</h6>
```

**Review by HassanBahati — COMMENTED** [review]
@basiima please find the suggestions. 

well done on the contributions :clap:

**Comment by basiima** [comment]
Thanks @HassanBahati . I've made the necessary updates based on your review

**Comment by HassanBahati** [comment]
> Thanks @HassanBahati . I've made the necessary updates based on your review

awesome. however i have made some changes in speaker section 

- just formatted the file
- then also there was a bug...we were having 2 `className` atrributes to the `<h6>`

**Review by HassanBahati — APPROVED** [review]
looks good

**Review by Amanya-Brian — APPROVED** [review]
Good to go

**Pull request merged**